WordPress/playground-tools
# WP-NOW: Use rotatePHPRuntime to avoid memory error

- Related to https://github.com/WordPress/playground-tools/issues/96

## What?



It uses the new function `rotatePHPRuntime` to mitigate the memory error.

## How?

Update the dependencies and use `rotatePHPRuntime` for the main instance.



## Testing Instructions






* Run:
   * `nvm use && npm install`
   * `npx nx build wp-now`
   * `node dist/packages/wp-now/cli.js start`
   * Observe wp-now has started correctly
